Packs/CVSS/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "CVSS",
-    "description": "Tools associated with CVSS scores and calculations.",
+    "name": "CVSS  (Deprecated)“",
+    "description": "Deprecated. Use from CommonScripts instead.",
```

Comment:
What about hidden: true ? with all the deprecated  packs

Reply:
oh no